### PR TITLE
Update Emission to 1.4 beta 5

### PR DESCRIPTION
**Pull request by orta**
screen shot 2017-11-02 at 4 45 13 pm

Updates React Native to 0.48.4, and the Emission lib to 1.4.0.beta.5. Also includes Eigen-side docs on how to update Emission if needs be.

@artsy/mobile - this adds a new env key (for consignments' location look up ) you can find it in 1pass, or you can make it a blank string and it will get a throwaway value in emission

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Artsy/App/ARAppDelegate+Emission.m [hunk 1]
```diff
@@ -109,11 +110,18 @@ - (void)setupSharedEmissionWithPackagerURL:(NSURL *)packagerURL;
...
+
+
+    NSString *gravity = [[ARRouter baseApiURL] absoluteString];
+    NSString *metaphysics = [ARRouter baseMetaphysicsApiURLString];
+    AREmissionConfiguration *config = [[AREmissionConfiguration alloc] initWithUserID:userID
+                                                                  authenticationToken:authenticationToken
+                                                                            sentryDSN:sentryDSN
+                                                                     googleMapsAPIKey:[keys googleMapsAPIKey]
+                                                                          gravityHost:gravity
+                                                                      metaphysicsHost:metaphysics];
```
Comment (ashfurrow) [inline Artsy/App/ARAppDelegate+Emission.m]: Having worked in modern JS lately, it almost impresses me just how badly Xcode messes up indentation.

File: Podfile [hunk 2]
```diff
@@ -1,3 +1,10 @@
+using_bundler = defined? Bundler
+unless using_bundler
+  puts "\nPlease re-run using:".red
+  puts "  bundle exec pod install\n\n"
+  exit(1)
```
Comment (ashfurrow) [inline Podfile]: This is a nice and good addition 👍

File: Podfile [hunk 3]
```diff
@@ -24,7 +31,8 @@ plugin 'cocoapods-keys', {
     "AdjustProductionAppToken",
     "ArtsyEchoProductionToken",
     "SentryProductionDSN",
-    "SentryStagingDSN"
+    "SentryStagingDSN",
+    "GoogleMapsAPIKey",
```
Comment (ashfurrow) [inline Podfile]: What do you think about adding a comment with a description of what the key is used for? Ideally we could back-fill the rest (eventually).

**Comment by mennenia** [comment]
🙌

**Pull request merged**